core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -57,6 +57,8 @@ object ClientQuotaManagerConfig {
   val DefaultNumQuotaSamples = 11
   val DefaultQuotaWindowSizeSeconds = 1
   val MaxThrottleTimeSeconds = 30
+  // Purge sensors after 1 hour of inactivity
+  val inactiveSensorExpirationTimeSeconds  = 3600
```

Comment:
Capitalize. Also, do you think it is worth making this configurable? If not, do you think a lower default expiration value may be better? I think an hour is fine, but not sure.